client/flutter/helper-scripts/flutter-netlify-build.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
```

Comment:
Since we're removing Netlify, we should remove the `client/web` directory as well.

Reply:
will do